api/register.go

```diff
@@ -0,0 +1,42 @@
+package api
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/mhghw/fara-message/jwt"
+)
+
+type RegUser struct {
+	ID              string `json:"id"`
+	Username        string `json:"username"`
+	FirstName       string `json:"first_name"`
+	LastName        string `json:"last_name"`
+	Password        string `json:"password"`
+	ConfirmPassword string `json:"confirm_password"`
+	Gender          string `json:"gender"`
+	DateOfBirth     string `json:"date_of_birth"`
+}
+
+// handle func for register,receives a json string according to the RegUser struct and generate a JWT with ID parameter
```

Comment:
If you want to comment about what your function does, follow [godoc format](https://tip.golang.org/doc/comment)

Reply:
included in study plan....